src/main.rs

```diff
@@ -1,3 +1,7 @@
+extern crate structopt;
+#[macro_use]
+extern crate clap;
```

Comment:
I think we can eliminate these stay rust 2018 by importing the macros directly.

Reply:
I solved this here already: https://github.com/svenstaro/overpower/blob/master/src/main.rs